pymc/sampling.py

```diff
@@ -951,8 +961,9 @@ def _iter_sample(
         The number of samples to draw
     step : function
         Step function
-    start : dict, optional
-        Starting point in parameter space (or partial point). Defaults to model.initial_point if not (defaults to empty dict)
+    start : dict
+        Starting point in parameter space (or partial point).
+        Must be complete.
```

Comment:
Not entirely sure, but maybe we could be more explicit that "complete" (here and in the 2 other instances were "complete" is used in this PR)?